core/server/api/v0.1/themes.js

```diff
@@ -74,7 +74,7 @@ themes = {
             .then(() => {
                 return themeService.storage.setFromZip(zip);
             })
-            .then((theme) => {
+            .then(({theme}) => {
```

Comment:
This piece needs ` this.headers.cacheInvalidate = true;` as well, right?

Reply:
🤔 `this.headers` cache invalidation concept was only added from v2, v0.1 didn't have the cache invalidation or other headers in controller. Also I had a look at file history for v0.1 controller and it was not using or setting cache invalidation header anywhere for this case.

Reply:
Damn checked the history and saw it was added during theme v2 [migration](https://github.com/TryGhost/Ghost/commit/eafbaaeba53b8ef7513d1bd978a3c25765732db0), and it's not clear why :confused:

Reply:
Yeah not sure, but think v0.1 handles cache invalidation differently. 😬

Reply:
Yes, it does. Think it's ok to have 'same state' as there was previously in v0.1. Let's merge this in :ship: